ui.apps/src/main/content/jcr_root/apps/venia/components/commerce/minicart/v1/minicart/clientlib/js/MiniCart.js

```diff
@@ -131,7 +131,16 @@
                     this.cartData = result[0];
                     this.cartTotals = result[1];
 
-                    let productData = this.cartData.items.map(item => ({[item.name]: item.sku})).reduce((acc, item) => (Object.assign(acc, item)), {});
+                    let productData = {};
+
+                    this.cartData.items.forEach(item => {
+                        let itemData = productData[item.name];
+                        if (!itemData) {
+                            productData[item.name] = [item.sku];
```

Comment:
This was a bit confusing to me.

Reply:
See the comment above - because you can have multiple cart items with the same name (same product, different variants) we cannot use a simple `{productName:productSku}` map, so we have to use something in the form of `{productName:[list-of-skus]}`.